### ISSUE TITLE
./installer not recognizing Office in RDPWindows

### ISSUE THREAD

**Issue by lonestrangerr** [Fmstrat/winapps#193]
Hi, MS Office is installed in the following location in RDPWindows
C:\Program Files (x86)\Microsoft Office\root\Office16
./installer don't recognize this path. 

how can I changes svg.icon and info in Apps folder 

@Fmstrat please help out.

**Comment by DennisVozian** [comment]
i got that error before, check if you have the bin directory in /home/user/.local/ if you don't, create the bin directory and run the install.sh again.

**Comment by lonestrangerr** [comment]
@DennisVozian thank you so much. It worked like a charm. Appreciated. One quick question, is there possibility to add new program?. I want to add "Zotero" a citation manager.

**Comment by DennisVozian** [comment]
i think you can install zotero on linux natively.

**Comment by lonestrangerr** [comment]
Yes, It can be installed natively, but MS word won't recognize the Zotero addon then.

**Comment by DennisVozian** [comment]
if Zotero is supported as an add-on in the 32-bit version of word, just add it to word from withing the VM and you should be good to go.

**Comment by lonestrangerr** [comment]
Yes, but need to import reference from browser to zotero so both should be running too. I managed to solve this issue for now by running them manually as per  Installation instruction. Thank you @DennisVozian for helping out.

**Comment by Sebastiangperez** [comment]
Is this project abandoned?

**Comment by 89pterWam** [comment]
Guys,./installer.sh
? Would you like to install for the current user or the whole system? User
Removing any old configurations...
grep: /home/paul/.local/bin/*: No such file or directory
rm: cannot remove '/usr/local/share/winapps/apps': Permission denied
grep: /usr/local/bin/*: No such file or directory
Installing...
  Checking for installed apps in RDP machine (this may take a while)... Finished.
  Configuring Windows...tee: //home/paul/.local/bin/windows: No such file or directory
chmod: cannot access '/home/paul/.local/bin/windows': No such file or directory
 Finished.
? How would you like to handle WinApps pre-configured applications? Select which pre-configured applications to set up
? Which pre-configured apps would you like to set up? Command Prompt (cmd), Microsoft Powershell (powershell)
cp: cannot create regular file '/home/paul/.local/bin/winapps': No such file or directory
  Configuring Cmd...tee: /home/paul/.local/bin/cmd: No such file or directory
chmod: cannot access '/home/paul/.local/bin/cmd': No such file or directory
 Finished.
  Configuring Powershell...tee: /home/paul/.local/bin/powershell: No such file or directory
chmod: cannot access '/home/paul/.local/bin/powershell': No such file or directory
 Finished.
? How would you like to handle other detected applications? Select which applications to set up
? Which other apps would you like to set up? LicenseManagerShellExt (licensemanagershellext.exe)
  Configuring LicenseManagerShellExt...tee: /home/paul/.local/bin/licensemanagershellext.exe: No such file or directory
chmod: cannot access '/home/paul/.local/bin/licensemanagershellext.exe': No such file or directory
 Finished.
Installation complete.

I really need help here I followed all the process for installing VM and Winapps bu when am trying to run ./installer.sh here is the output:

**Comment by 89pterWam** [comment]
bin/winapps check
And after accepting the Certficate and try to the bin check,
This is what I ran into:

[06:37:48:967] [27053:27054] [WARN][com.freerdp.crypto] - Certificate verification failure 'self-signed certificate (18)' at stack position 0
[06:37:48:967] [27053:27054] [WARN][com.freerdp.crypto] - CN = RDPWindows
[06:37:49:472] [27053:27054] [INFO][com.freerdp.gdi] - Local framebuffer format  PIXEL_FORMAT_BGRX32
[06:37:49:472] [27053:27054] [INFO][com.freerdp.gdi] - Remote framebuffer format PIXEL_FORMAT_BGRA32
[06:37:49:475] [27053:27054] [INFO][com.freerdp.channels.rdpsnd.client] - [static] Loaded fake backend for rdpsnd
[06:37:49:475] [27053:27072] [INFO][com.freerdp.channels.rdpdr.client] - Loading device service drive [home] (static)
[06:37:49:475] [27053:27054] [INFO][com.freerdp.channels.drdynvc.client] - Loading Dynamic Virtual Channel rdpgfx
[06:37:49:475] [27053:27054] [INFO][com.freerdp.channels.drdynvc.client] - Loading Dynamic Virtual Channel disp
[06:37:50:410] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:50:416] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:50:420] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:50:420] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:50:434] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:50:444] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:50:576] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:50:587] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:51:550] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:52:999] [27053:27072] [INFO][com.freerdp.channels.rdpdr.client] - registered device #1: home (type=8 id=1)
[06:37:52:212] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:52:231] [27053:27054] [WARN][com.freerdp.client.x11] - xf_Pointer: Invalid appWindow
[06:37:52:348] [27053:27054] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:52:379] [27053:27054] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:52:409] [27053:27054] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:52:415] [27053:27054] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:52:416] [27053:27054] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:52:416] [27053:27054] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:52:417] [27053:27054] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:062] [27053:27054] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:073] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:075] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:104] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:106] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:131] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:133] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:168] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:169] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:217] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:218] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:255] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:255] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:288] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:288] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:474] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:37:53:474] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:06:788] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:06:788] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:06:860] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:06:860] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:193] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:193] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:255] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:255] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:297] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:297] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:379] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:379] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:411] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:411] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:442] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:07:443] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:08:758] [27053:27054] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:08:855] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:08:855] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:08:855] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:12:817] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:12:817] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:12:817] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:24:109] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:24:109] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:24:109] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:24:281] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:24:281] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
[06:38:24:285] [27053:27076] [WARN][com.freerdp.client.x11] - xf_lock_x11_:	[1] recursive lock from xf_UpdateWindowArea
^C[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - Caught signal 'Interrupt' [2]
[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 0: /lib/x86_64-linux-gnu/libwinpr2.so.2(winpr_backtrace+0x54) [0x7f403c2790b4]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 1: /lib/x86_64-linux-gnu/libwinpr2.so.2(winpr_log_backtrace_ex+0x33) [0x7f403c27bd53]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 2: /lib/x86_64-linux-gnu/libfreerdp2.so.2(+0x2c844) [0x7f403c43c844]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 3: /lib/x86_64-linux-gnu/libc.so.6(+0x42520) [0x7f403be42520]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 4: /lib/x86_64-linux-gnu/libc.so.6(__poll+0x4f) [0x7f403bf18bcf]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 5: /lib/x86_64-linux-gnu/libwinpr2.so.2(+0x3d370) [0x7f403c266370]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 6: /lib/x86_64-linux-gnu/libwinpr2.so.2(WaitForSingleObjectEx+0x4eb) [0x7f403c266c1b]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 7: xfreerdp(+0x864b) [0x55f76666764b]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 8: /lib/x86_64-linux-gnu/libc.so.6(+0x29d90) [0x7f403be29d90]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 9: /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80) [0x7f403be29e40]

[06:41:38:495] [27053:27053] [ERROR][com.freerdp.utils] - 10: xfreerdp(_start+0x25) [0x55f7666676f5]
